ui/redux/selectors/stripe.js

```diff
@@ -27,6 +27,17 @@ export const selectAccountDefaultCurrency = (state: State) => {
 };
 
 export const selectAccountUnpaidBalance = (state: State) => selectAccountStatus(state)?.total_received_unpaid || 0;
+
+export const selectAccountId = (state: State) => {
+  const accountStatus = selectAccountStatus(state);
+  return accountStatus && accountStatus.account_info?.id;
+};
+
+export const selectAccountEmail = (state: State) => {
+  const accountStatus = selectAccountStatus(state);
+  return accountStatus && accountStatus.account_info?.email;
+};
+
```

Comment:
:small_blue_diamond: Need to chill with the selectors :)

No changes needed, but one thing I've noticed from memberships and rentals is we are going too far with selectors, especially those without any transformation logic.

For this case, `selectAccountInfo` is probably fine to encapsulate access for one level down, and that could be used instead `selectAccountId` and `selectAccountEmail`.  Just define the type properly (instead of `any`) and the code is just a safe/readable.

<sub>https://redux.js.org/usage/deriving-data-selectors#balance-selector-usage</sub>
<img width="760" alt="image" src="https://user-images.githubusercontent.com/64950861/196750881-037978e0-4034-459c-b164-831daf82323a.png">